roles/st2chatops/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+# st2chatops version to install. Use `latest` to get automatic updates or pin it to numeric version like `2.2.0`
+st2chatops_version: latest
+
+# Please provide ST2_API_KEY using "st2 apikey create -k"
+st2chatops_ST2_API_KEY: "CHANGE-ME-PLEASEdd"
```

Comment:
Same as previous comment, just an empty string or maybe `None` (which won't set the val in st2chatops.env) instead of `CHANGE-ME-PLEASEdd` will look nicer.